WooCommerce/Classes/Extensions/ViewControllerPresenting.swift

```diff
@@ -8,6 +8,7 @@ protocol ViewControllerPresenting: AnyObject {
     func dismiss(animated: Bool, completion: (() -> Void)?)
     func dismiss(animated: Bool)
     func show(_ vc: UIViewController, sender: Any?)
+    var presentingViewController: UIViewController? { get }
```

Comment:
```suggestion
```

We can remove this property from here, the `NullViewControllerPresenting`, and the `MockViewControllerPresenting`, right? The dismissal bit for modal presentation seems to still work fine when removed as it's still part of the `UIViewController` class, and we're not using it for testing:

```
if let presentingViewController = onboardingViewController.presentingViewController {
    presentingViewController.dismiss(animated: true)
}
```

TIL about `presentingViewController`, I was curious how it was assigned since I couldn't see it in our code 👍

Reply:
Thanks for spotting this. Yes, this was left over from an earlier attempt where I thought we needed to do it at a different part of the flow. Removed in 367731c